src/libraries/System.Diagnostics.Process/src/Microsoft/Win32/SafeHandles/SafeProcessHandle.cs

```diff
@@ -19,7 +19,7 @@ public sealed partial class SafeProcessHandle : SafeHandleZeroOrMinusOneIsInvali
     {
         internal static readonly SafeProcessHandle InvalidHandle = new SafeProcessHandle();
 
-        internal SafeProcessHandle()
+        public SafeProcessHandle()
```

Comment:
GitHub won't let me comment on it, but we should be able to delete methods like InitialSetHandle below and just use the new Marshal method.